statics/js/components/preference.js

```diff
@@ -0,0 +1,142 @@
+/* jshint multistr: true */
+
+var PreferenceComponent = {
+
+  name: 'Preference',
+
+  mixins: [apiMixin, notificationMixin],
+
+  template: '\
+    <form class="form-preference" @submit.prevent="save">\
+      <div class="form-group">\
+        <label for="default-filter">Filters</label>\
+        <a><i class="fa fa-question help-text" aria-hidden="true" title="filter and highlight gremlin queries, displayed in the left panel"></i></a>\
+        <div v-for="filter in filters">\
+          <div class="form-group">\
+            <div class="input-group">\
+              <label class="input-group-addon">Name: </label>\
```

Comment:
We should add a "delete" button right to this field in order to allow the user to delete previously defined filter

Reply:
nice idea! added a button to delete.